shared/chat/selectable-big-team-channel.js

```diff
@@ -73,29 +83,34 @@ export const rowHeight = isMobile ? 64 : 56
 
 const styles = Styles.styleSheetCreate({
   channelname: Styles.platformStyles({
-    common: {flexBasis: '70%'},
+    common: {
+      flexBasis: '70%',
+      flexShrink: 0.3,
```

Comment:
why this change

Reply:
flexBasis didn't work well. On desktop, if the team name is too long, then we'd end up having channel name taking 70% space that has a bunch of white space. On mobile if team name is too long, we'd have channel name pushed to far right. Looks like flexShrink doesn't work perfectly either, but I'll play with it and see

Reply:
OK I think I've found a way to make it work, by using maxWidth and flexShrink. The result is that channelname only grows if it's overflowing, but only up to 70%. If within 70% channelname can show in full, we just shrink teamname until they fit. If both needs shrinking, then it's 3:7 between teamname and channel name. Here are some screenshots:

![image](https://user-images.githubusercontent.com/255797/51571388-e6d12e00-1e56-11e9-959e-6cfaef9c06a4.png)

![image](https://user-images.githubusercontent.com/255797/51571398-eb95e200-1e56-11e9-8d96-fc26b040f9b1.png)